app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,16 @@
 class PostsController < ApplicationController
   # Placeholder action for all posts by a given user
   def index
-    # Your logic here
+    @posts = Post.all
+    @placeholder_text = 'Placeholder text for all posts by a given user'

```

Comment:
**OPTIONAL** - Good job done so far, it's not good practice to populate your controller with items that are not needed, please try to keep content in your controller as minimal as possible. I suggest you render this in the respective view file 👍 👍